spec.bs

```diff
@@ -364,6 +364,7 @@ const request = new PaymentRequest([{
     instrument: {
       displayName: "Fancy Card ****1234",
       icon: "https://fancybank.com/card-art.png",
+      iconMustBeShown: true,
```

Comment:
I think we don't want to include this in the example, for two reasons:

1. Setting this to `true` is equivalent to not including it, so doesn't seem useful to include.
2. But setting it to `false` might run us afoul of people copy/pasting examples, so we probably don't want to do that either!

For now I suggest we remove this line, and we can talk later about whether it should be included in a separate example.

Reply:
Done